incubation.md

```diff
@@ -0,0 +1,130 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which certain types of new Jupyter Subprojects are created and
+evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a
+GitHub repository with a development team that follows the Jupyter governance and contribution
+model.
+
+The incubation process doesn't apply to all new Subprojects, only those with the following
+characteristics:
+
+* Significant unanswered technical questions that required exploration.
+* Entirely new directions, scopes or ideas that haven't been vetted with the community.
+* Significant, already existing code bases where it is not clear how the project will
+  integrate with the rest of Jupyter.
+ 
+Most importantly, the incubation process does not apply to new Subprojects that are
+created in the natural development of existing officially supported Subproject in the
+main jupyter GitHub organization.
```

Comment:
Thanks for this bit, I think it clarifies where incubation sits in the scheme of things.